### PR TITLE
Add term entry for semantic html ruby

### PR DESCRIPTION
**Pull request by droffilc1**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/html/concepts/semantic-html/terms/ruby/ruby.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  - 'Web Design'
+Tags:
+  - 'Semantic'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`<ruby>`** tag is a semantic HTML element that represents small annotations usually used for showing pronunciation of Japanese and East Asian characters. You can use `<ruby>` together with `<rt>` /or `<rp>`. The `<ruby>` entails one or more chacters which require an explanation or pronunciation, and an `<rt>` element that provides that information, and an optional `<rp>` element which ascertains what to show to browsers that lack support for annotations.
```
Comment (yangc95) [inline content/html/concepts/semantic-html/terms/ruby/ruby.md:15]: Remove pronouns (I we you) and keep language definitive, I.e. Use <this> to accomplish <that>.

File: content/html/concepts/semantic-html/terms/ruby/ruby.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Syntax
+
+```html
+<ruby>
+  <!-- contents go here -->
```
Comment (yangc95) [inline content/html/concepts/semantic-html/terms/ruby/ruby.md:21]:
```suggestion
  <!-- Contents go here -->
```

**Review by yangc95 — COMMENTED** [review]
@droffilc1 awesome! I left some minor comments. If you're up for it, could you whip up an image for the examples? Let me know if you need any assistance

**Comment by yangc95** [comment]
> @yangc95 I need assistance to whip up an image for the examples

@droffilc1 sure! How's this:

Screenshot 2023-08-10 at 3 56 24 PM

I'm thinking we can also group the examples together into one example.

**Comment by droffilc1** [comment]
> > @yangc95 I need assistance to whip up an image for the examples
> 
> @droffilc1 sure! How's this:
> 
> Screenshot 2023-08-10 at 3 56 24 PM
> 
> I'm thinking we can also group the examples together into one example.

It looks great

File: content/html/concepts/semantic-html/terms/ruby/ruby.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+<ruby> ゆきひろ <rt> Yukihiro </rt> </ruby>
+```
+
+## Example 2
+
+This snippet code shows `<ruby>` being used with `<rt>` and `<rp>` tag:
+
+```html
+<ruby> ゆきひろ まつもと <rp>(</rp><rt>Yukihiro Matsumoto</rt><rp>)</rp> </ruby>
+```
```
Comment (yangc95) [inline content/html/concepts/semantic-html/terms/ruby/ruby.md:39]: Can you combine these into one example and change the image name to `semantic-html-ruby.png`

Comment (droffilc1) [reply content/html/concepts/semantic-html/terms/ruby/ruby.md:39]: Okay

File: content/html/concepts/semantic-html/terms/ruby/ruby.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  <!-- Contents go here -->
+</ruby>
+```
+
+## Example
+
+This snippet code shows `<ruby>` being used with `<rt>` and `<rp>` tag:
+
+```html
+<ruby> ゆきひろ まつもと <rp>(</rp><rt>Yukihiro Matsumoto</rt><rp>)</rp> </ruby>
```
Comment (yangc95) [inline content/html/concepts/semantic-html/terms/ruby/ruby.md:30]:
```suggestion
<ruby> ゆきひろ まつもと
  <rp>(</rp><rt>Yukihiro Matsumoto</rt><rp>)</rp>
</ruby>
```

File: content/html/concepts/semantic-html/terms/ruby/ruby.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+```
+
+This code results in the following display:
+![Output of the ruby tag example](https://raw.githubusercontent.com/Codecademy/docs/main/ruby/semantic-html-ruby.png)
```
Comment (yangc95) [inline content/html/concepts/semantic-html/terms/ruby/ruby.md:34]:
```suggestion

![HTML ruby tag example](https://raw.githubusercontent.com/Codecademy/docs/main/ruby/semantic-html-ruby.png)
```

**Review by yangc95 — APPROVED** [review]
@droffilc1 thanks for making those changes! Looks good

**Review by KTom101 — CHANGES_REQUESTED** [review]
@droffilc1 I added a few adjustments.

**Review by KTom101 — APPROVED** [review]
@droffilc1 I added a couple more changes directly. Otherwise, LGTM 👍

**Comment by yangc95** [comment]
@droffilc1 Would you be able to open up a pull request to edit the image url? Remove "/ruby" from that url.

**Comment by droffilc1** [comment]
> @droffilc1 Would you be able to open up a pull request to edit the image url? Remove "/ruby" from that url.

Yes!

**Comment by yangc95** [comment]
@droffilc1 thanks! If you need any assistance please let me know as well

**Comment by droffilc1** [comment]
@yangc95 I have made the changes. Waiting for this PR to be reopened

**Comment by yangc95** [comment]
@droffilc1 Oh, I see - can you open a new PR with those changes?

**Comment by droffilc1** [comment]
@yangc95 I just did.

**Pull request merged**